WordPress/src/main/java/org/wordpress/android/ui/blaze/ui/blazeoverlay/BlazeViewModel.kt

```diff
@@ -0,0 +1,69 @@
+package org.wordpress.android.ui.blaze.ui.blazeoverlay
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import dagger.hilt.android.lifecycle.HiltViewModel
+import org.wordpress.android.ui.blaze.BlazeFeatureUtils
+import org.wordpress.android.ui.blaze.BlazeFlowSource
+import org.wordpress.android.ui.blaze.BlazeUiState
+import org.wordpress.android.ui.blaze.PostUIModel
+import java.util.Locale
+import javax.inject.Inject
+
+@HiltViewModel
+class BlazeViewModel @Inject constructor(private val blazeFeatureUtils: BlazeFeatureUtils) : ViewModel() {
+    private lateinit var blazeFlowSource: BlazeFlowSource
+
+    private val _refreshAppTheme = MutableLiveData<Unit>()
+    val refreshAppTheme: LiveData<Unit> = _refreshAppTheme
+
+    private val _refreshAppLanguage = MutableLiveData<String>()
+    val refreshAppLanguage: LiveData<String> = _refreshAppLanguage
```

Comment:
np: Reminder to remove this if `LocaleAwareActivity` renders `LocaleAwareComposable` useless for this screen.

Reply:
Yup removed in 4b4d04f7550fda5cafa3b50a10d372ac0d6337e7 and removed the imports in 10e934ff80959081c99296c241f427bd4cf9d671